src/requirements/data/majors/cs.ts

```diff
@@ -105,18 +105,33 @@ const csRequirements: readonly CollegeOrMajorRequirement[] = [
       'Frequently, the three courses are from the same department.',
     source:
       'https://www.cs.cornell.edu/undergrad/rulesandproceduresengineering/choosingyourelectives',
-    fulfilledBy: 'self-check',
-    minCount: 3,
+    checker: [
+      (course: Course): boolean => {
+        const { catalogNbr } = course;
+        return !(ifCodeMatch(catalogNbr, '1***') || ifCodeMatch(catalogNbr, '2***'));
+      },
+    ],
+    checkerWarning: 'We do not check that the courses are related.',
+    fulfilledBy: 'courses',
+    perSlotMinCount: [3],
   },
+  // TODO: Doesn't check for ROTC (PE) above 3000
   {
     name: 'Major-approved Elective(s)',
     description:
       'At least 3 credit hours total. All academic courses count.' +
       'No PE courses, courses numbered 10xx, and ROTC courses below the 3000-level allowed.',
     source:
       'https://www.cs.cornell.edu/undergrad/rulesandproceduresengineering/choosingyourelectives',
-    fulfilledBy: 'self-check',
-    minCount: 3,
+    checker: [
+      (course: Course): boolean => {
+        const { catalogNbr } = course;
+        return !(ifCodeMatch(course.subject, 'PE') && ifCodeMatch(catalogNbr, '10**'));
+      },
+    ],
```

Comment:
A note on this. Later we should build a special case on these kind of accept all requirements to avoid bloating the requirement json

Reply:
i'll add to notion,ty!